data/resources.json

```diff
@@ -57,6 +57,10 @@
         {
           "title": "Swagger",
           "value": "https://lcd-axelar.imperator.co/static/swagger/"
+        },
+        {
+          "title": "Swagger",
+          "value": "https://lcd-axelar.stakerun.com/static/swagger/"
```

Comment:
Can you change the titles so that it's clear the difference?

Reply:
sure